src/qt/utilitydialog.cpp

```diff
@@ -27,82 +29,117 @@
-    } else {
-        setWindowTitle(tr("Command-line options"));
-        QString header = "Usage:  bitcoin-qt [command-line options]                     \n";
-        QTextCursor cursor(ui->helpMessage->document());
-        cursor.insertText(version);
-        cursor.insertBlock();
-        cursor.insertText(header);
-        cursor.insertBlock();
-
-        std::string strUsage = gArgs.GetHelpMessage();
-        QString coreOptions = QString::fromStdString(strUsage);
-        text = version + "\n\n" + header + "\n" + coreOptions;
-
-        QTextTableFormat tf;
-        tf.setBorderStyle(QTextFrameFormat::BorderStyle_None);
-        tf.setCellPadding(2);
-        QVector<QTextLength> widths;
-        widths << QTextLength(QTextLength::PercentageLength, 35);
-        widths << QTextLength(QTextLength::PercentageLength, 65);
-        tf.setColumnWidthConstraints(widths);
-
-        QTextCharFormat bold;
-        bold.setFontWeight(QFont::Bold);
-
-        for (const QString &line : coreOptions.split("\n")) {
-            if (line.startsWith("  -"))
-            {
-                cursor.currentTable()->appendRows(1);
-                cursor.movePosition(QTextCursor::PreviousCell);
-                cursor.movePosition(QTextCursor::NextRow);
-                cursor.insertText(line.trimmed());
-                cursor.movePosition(QTextCursor::NextCell);
-            } else if (line.startsWith("   ")) {
-                cursor.insertText(line.trimmed()+' ');
-            } else if (line.size() > 0) {
-                //Title of a group
-                if (cursor.currentTable())
+        case AboutMode:
+        {
+            setWindowTitle(tr("About %1").arg(PACKAGE_NAME));
+
+            std::string licenseInfo = LicenseInfo();
+            /// HTML-format the license message from the core
+            QString licenseInfoHTML = QString::fromStdString(LicenseInfo());
+            // Make URLs clickable
+            QRegularExpression uri(QStringLiteral("<(.*)>"), QRegularExpression::InvertedGreedinessOption);
+            licenseInfoHTML.replace(uri, QStringLiteral("<a href=\"\\1\">\\1</a>"));
+            // Replace newlines with HTML breaks
+            licenseInfoHTML.replace("\n", "<br>");
+
+            ui->aboutMessage->setTextFormat(Qt::RichText);
+            ui->scrollArea->setVerticalScrollBarPolicy(Qt::ScrollBarAsNeeded);
+            text = version + "\n" + QString::fromStdString(FormatParagraph(licenseInfo));
+            ui->aboutMessage->setText(version + "<br><br>" + licenseInfoHTML);
+            ui->aboutMessage->setWordWrap(true);
+            ui->helpMessage->setVisible(false);
+            ui->labelTxId->setVisible(false);
+            ui->txidEdit->setVisible(false);
+            ui->labelBlockHash->setVisible(false);
+            ui->blockHashEdit->setVisible(false);
+            ui->submitButton->setVisible(false);
+            ui->verboseCheckbox->setVisible(false);
+            break;
```

Comment:
Since you are here (refactoring commit 10b3cd129e1fa8ece229a255d6fff89f419751c0) I'd extract all these settings into a function like "`initializeAboutModeWindow`" or something like that, same for the other modes, making the code a bit more maintainable and easier to follow.